in/generateNetLinux.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+"$SUMO_HOME"/bin/netgenerate --rand --random -o example.net.xml --rand.iterations=200 --roundabouts.guess=true --sidewalks.guess=true --crossings.guess=true --tls.guess=true --tls.guess.threshold=50
```

Comment:
only putting quotes around the environment variable is unusuable, but valid. I would have done it like this
```suggestion
"$SUMO_HOME/bin/netgenerate" --rand --random -o example.net.xml --rand.iterations=200 --roundabouts.guess=true --sidewalks.guess=true --crossings.guess=true --tls.guess=true --tls.guess.threshold=50
```

Reply:
Okay. I will change that 👍